core/modules/layout/plugins/renderers/layout_renderer_standard.inc

```diff
@@ -523,12 +483,19 @@ class LayoutRendererStandard {
         $title = $block->getTitle();
       }
 
-      // Assemble a structured content array.
-      $content = (object) array(
-        'title' => $title,
+      $data = array(
+        'subject' => $title,
         'content' => $content,
       );
 
+      // Add a backwards compatible way to alter all blocks.
+      backdrop_alter(array('block_view', 'block_view_' . $block->module . '_' . str_replace('-', '_', $block->delta)), $data, $block);
+
+      $content = (object) array(
+        'title' => $data['subject'],
+        'content' => $data['content'],
+      );
```

Comment:
The comment `// Add a backwards compatible way to alter all blocks.` should remove "backwards compatible", as this is actually an explicit API change from Drupal 7.

As this is an API change, perhaps we should change the `$data` that is passed into `hook_block_alter()` to use "title" instead of "subject"? Then *only* legacy blocks using `hook_block_view()` would still be using "subject".